rpa-builder/6.7/modules/ROOT/pages/toolbox-excel-operations-execute-excel-macro.adoc

```diff
@@ -0,0 +1,62 @@
+= Execute Excel Macro 
+
+Run a macro in an Excel file or sheet. You can specify and pass up to 30 variables to the macro.
+
```

Comment:
We should add the info, that this action step can only be placed outside of the Excel Session and needs an Microsoft Excel installation on the system to function. Both facts different from the previous Excel Action Steps.
When running, the action step will check and throw an error if no Excel application installation was found.